server/db/db.js

```diff
@@ -1,6 +1,7 @@
 const { Pool } = require('pg');
 const dotenv = require('dotenv');
 const fs = require('fs');
+const migrationUtils = require('../api/actions/migrations');
```

Comment:
you're only using one function from here, so might as well destructure it:

```suggestion
const { getOutStandingMigrations } = require('../api/actions/migrations');
```